tooling/minimize/src/main.rs

```diff
@@ -67,9 +69,15 @@ pub use miniutil::fmt::dump_program;
 pub use miniutil::run::*;
 
 // Get back some `std` items
+pub use std::env;
 pub use std::format;
+use std::path::PathBuf;
+pub use std::process::Command;
 pub use std::string::String;
```

Comment:
There should only be imports here that need to shadow glob imports above. Do the new ones truly need that? If not, please move them down to "imports for main".